.travis.yml

```diff
@@ -36,14 +23,18 @@ matrix:
       env: SYMFONY_VERSION=4.0.*;ASSETIC=skip
     - php: 7.3
       env: SYMFONY_VERSION=4.1.*;ASSETIC=skip
+    - php: 7.3
+      env: SYMFONY_VERSION=4.2.*;ASSETIC=skip
     - php: 7.3
       env: SYMFONY_VERSION=@dev;ASSETIC=skip
     - php: 7.0
-      env: TWIG_VERSION=2.7.*
-    - php: 7.1
+      env: SYMFONY_VERSION=3.0.*;TWIG_VERSION=2.4.*
```

Comment:
why forcing to use the unmaintained 3.0 version of Symfony ?

Reply:
because its the lower bound in the composer.json